content/academy/data_collection_basics/using_devtools.md

```diff
@@ -32,72 +32,73 @@ The function to do that is called `document.querySelector('some-selector')` and
+The `document.querySelector()` function looks for a specific element in the whole HTML `document`, so if we called it with `h3`, it would find the first `<h3>` node in the `document`. Luckily we can also use this function to look for elements within an element.
 
-There's even a similar function called <a href="https://javascript.info/searching-elements-dom#querySelectorAll" target="_blank">`querySelectorAll()`</a> that returns an array of all the elements matching the selector, not only the first one. We will use that function to get all the fields with data.
+There's a similar method called <a href="https://javascript.info/searching-elements-dom#querySelectorAll" target="_blank">`querySelectorAll()`</a>, which returns an array of all elements matching a specified selector - not just the first one. We will use this method to grab all the elements holding our sought after data.
 
 > Learn more about `Arrays` <a href="https://javascript.info/array" target="_blank">in this tutorial</a>.
 
-Don't forget that earlier we saved the parent element into a variable called `item`. We can use that variable to search only within that element.
+Let's find the parent element of all of the products, which matches the selector `div.grid.gap-6`, select it with `document.querySelector()`, then find all of the product elements.
+
+```JavaScript
+const grid = document.querySelector('div.grid.gap-6');
+
+const products = grid.querySelectorAll('a[href*="/product/"]')
 
-```js
-const fields = item.querySelectorAll('div.td');
-console.log(fields);
+console.log(products);
 ```
 
-![List child elements with JavaScript]({{@asset data_collection_basics/images/list-child-elements.webp}})
+There are 32 products on the page, so if we've done this correctly, a list of 32 product elements should be logged to the console.
+
+![List child elements with JavaScript]({{@asset data_collection_basics/images/list-child-elements-new.png}})
 
 ### [](#collecting-data) Collecting data
 
-The `fields` array now contains all the elements we need and we can access the data individually. Let's save the results into an <a href="https://javascript.info/object" target="_blank">object</a>. Those of you who know JavaScript will know that this is not the prettiest code ever written, but it is beginner-friendly and that's important here. We will also use the `.trim()` function to remove unnecessary whitespace from the results.
+The `products` array now contains all the elements we need, and we can access each one's data individually. Let's save the title and price of the first product into an <a href="https://javascript.info/object" target="_blank">object</a>. Those of you who know JavaScript will know that this is not the prettiest code ever written, but it is beginner-friendly and that's important here. We will also use the `.trim()` method to remove unnecessary whitespace from the results.
```

Comment:
AFAIK this part of the tutorial was about getting data for one website, so it should be about one product. The next part deals with getting data for more products.

Reply:
If you check the end, we are grabbing only the data for one single product. The reason we are selecting ALL products here is to demonstrate querySelectorAll